<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cube_ans</class>
 <widget class="QMainWindow" name="cube_ans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>cube_ans</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Slice</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="verticalSpacing">
         <number>1</number>
        </property>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sliceSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Wavelengt Value</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="sliceMaximumValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2" colspan="2">
         <widget class="QSlider" name="sliceSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="wavelengthLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Slice Value</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="sliceMinimumValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QGroupBox" name="spaceCubePlot">
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUnselect"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionZoom"/>
    <addaction name="actionPan"/>
    <addaction name="actionZoom_reset"/>
    <addaction name="separator"/>
    <addaction name="actionRectangle_coordinates"/>
    <addaction name="actionCreation_Rectangle"/>
    <addaction name="separator"/>
    <addaction name="actionEllipse_coordinates"/>
    <addaction name="actionCreation_Ellipse"/>
    <addaction name="separator"/>
    <addaction name="actionSpectrum_visualization"/>
    <addaction name="separator"/>
    <addaction name="actionBackground_subtraction"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Style</string>
    </property>
    <widget class="QMenu" name="menuColor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="actionGray"/>
     <addaction name="actionAccent"/>
     <addaction name="actionHeat"/>
     <addaction name="actionRainbow"/>
     <addaction name="actionCoolWarm"/>
    </widget>
    <widget class="QMenu" name="menuStretch">
     <property name="title">
      <string>Stretch</string>
     </property>
     <addaction name="actionSqrt"/>
     <addaction name="actionLog"/>
     <addaction name="actionLinear"/>
    </widget>
    <widget class="QMenu" name="menuScale">
     <property name="title">
      <string>Scale</string>
     </property>
     <addaction name="actionMinMax_Interval"/>
     <addaction name="actionZScale_Interval"/>
    </widget>
    <addaction name="menuColor"/>
    <addaction name="menuScale"/>
    <addaction name="menuStretch"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuStyle"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionMinMax_Interval">
   <property name="text">
    <string>MinMax Interval</string>
   </property>
  </action>
  <action name="actionZScale_Interval">
   <property name="text">
    <string>ZScale Interval</string>
   </property>
  </action>
  <action name="actionSqrt">
   <property name="text">
    <string>Sqrt</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionLinear">
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionAccent">
   <property name="text">
    <string>Accent</string>
   </property>
  </action>
  <action name="actionHeat">
   <property name="text">
    <string>Heat</string>
   </property>
  </action>
  <action name="actionRainbow">
   <property name="text">
    <string>Rainbow</string>
   </property>
  </action>
  <action name="actionCoolWarm">
   <property name="text">
    <string>CoolWarm</string>
   </property>
  </action>
  <action name="actionRectangle_coordinates">
   <property name="text">
    <string>Rectangle coordinates</string>
   </property>
  </action>
  <action name="actionCreation_Rectangle">
   <property name="text">
    <string>Creation of Rectangle parameterized</string>
   </property>
  </action>
  <action name="actionSpectrum_visualization">
   <property name="text">
    <string>Spectrum visualization</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionBackground_subtraction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Background subtraction</string>
   </property>
  </action>
  <action name="actionUnselect">
   <property name="text">
    <string>Unselect current tool</string>
   </property>
  </action>
  <action name="actionCreation_Ellipse">
   <property name="text">
    <string>Creation of Ellipse parameterized</string>
   </property>
  </action>
  <action name="actionZoom_reset">
   <property name="text">
    <string>Zoom reset</string>
   </property>
  </action>
  <action name="actionEllipse_coordinates">
   <property name="text">
    <string>Ellipse coordinates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
